qcodes/instrument/parameter.py

```diff
@@ -2283,9 +2292,9 @@ def __init__(self,
                  unit: str = None) -> None:
         # Set the name
         if name:
-            self.name = name
+            self._short_name = name
         else:
-            self.name = "{}_scaled".format(output.name)
+            self._short_name = "{}_scaled".format(output.name)
```

Comment:
so for this particular parameter the self.name/label/unit assignments seem to be done by mistake because anyway the values are passed to super.init below, and the super will take care of assigning it properly. so i suggest that this is fixed (here, or in a separate PR), so that "making name a readonly property" does not have impact on this parameter class.

Reply:
This is fixed now.